.github/dependabot.yml

```diff
@@ -29,9 +29,23 @@ updates:
         dependency-type: "production"
         update-types:
         - "major"
-      # Development dependencies
-      dev-dependencies:
-        dependency-type: "development"
+      # ESLint related dependencies
+      dev-dependencies-eslint:
```

Comment:
I plan to provides three groups, eslint, typescript and ajv because these three will failed when the core is bumped.
For the others, it is good to go with a single PR per package.

Reply:
Looks a lot better, thanks @climba03003!